bin/reparse

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+lib = File.expand_path("../../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+$VERBOSE = nil
+
+require "bundler/setup"
+
+require "ruby-next/language"
+
+contents =
+  if File.exist?(ARGV[0])
+    File.read(ARGV[0])
+  else
+    ARGV[0]
+  end
+
+ast = RubyNext::Language.parse(contents)
+puts Unparser.unparse(ast)
```

Comment:
You could reuse the `Unparser::Verification` class here to print more detials equivalent to the ones `unparser` CLI will.

Reply:
I'm using a custom parser, and, as far as I can see, Unparser has its own.

Thus, I'm using this script to verify that Unparser correctly works with my parser.

Reply:
its easy to change `Unparser::Verification` to accept a custom parser. That class produces a nicer report.